nim/related.nimble

```diff
@@ -13,3 +13,11 @@ bin           = @["related"]
 requires "nim >= 2.0.0"
 requires "jsony"
 requires "xxhash"
+
+task buildopt, "build optimized":
+  exec "nimble -d:profileGen -d:release build"
+  echo "Gen stat"
+  exec "./related > /dev/null"
+  exec "llvm-profdata-12 merge default.profraw --output default.profdata"
```

Comment:
is llvm12 needed

Reply:
Sorry, it is just my old ubuntu, will fix in a minute